packages/x-charts/src/hooks/useAxis.ts

```diff
@@ -6,35 +6,66 @@ import {
 } from '../internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxis.selectors';
 import { useSelector } from '../internals/store/useSelector';
 import { useStore } from '../internals/store/useStore';
+import { AxisId } from '../models/axis';
 
+/**
+ * Get all the X axes.
```

Comment:
I think that's how we standardize the docs naming for x/y axes
```suggestion
 * Get all the x-axes.
```